SO_properties.py

```diff
@@ -565,16 +564,43 @@ def spin_parameter(self) -> unyt.unyt_quantity:
     def TotalInertiaTensor(self) -> unyt.unyt_array:
         """
         Inertia tensor of the total mass distribution.
+        Computed iteratively using an ellipsoid with volume equal to that of
+        a sphere with radius SORadius. Only considers particles within the SORadius
         """
         if self.Mtotpart == 0:
             return None
-        return get_inertia_tensor(self.mass, self.position)
+        return get_inertia_tensor(self.mass, self.position, self.SO_r)
 
     @lazy_property
-    def ReducedTotalInertiaTensor(self):
+    def TotalInertiaTensorReduced(self) -> unyt.unyt_array:
+        """
+        Reduced inertia tensor of the total mass distribution.
+        Computed iteratively using an ellipsoid with volume equal to that of
+        a sphere with radius SORadius. Only considers particles within the SORadius
+        """
+        if self.Mtotpart == 0:
+            return None
+        return get_inertia_tensor(self.mass, self.position, self.SO_r, reduced=True)
+
+    @lazy_property
+    def TotalInertiaTensorNoniterative(self) -> unyt.unyt_array:
+        """
+        Inertia tensor of the total mass distribution.
+        Computed in a single iteration using particles within the SORadius.
```

Comment:
Might be clearer if you add that one iteration is equivalent to using particles within the SO sphere, without deforming the volume into an ellipsoid. Some people may assume this step is iteration 0 (i.e. you do no update the shape of volume), rather than iteration 1.